cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from .views import movie_list, movie_detail
+
+
+app_name = "cinema"
+
+urlpatterns = [
+    path("movies/", movie_list ,name="movie_list"),
+    path("movies/<int:pk>", movie_detail, name="movie_detail"),

```

Comment:
Consider adding a trailing slash to the `movie_detail` path (`"movies/<int:pk>/"`) to maintain consistency with the `movie_list` path and follow RESTful URL conventions.